data/wanderers start.txt

```diff
@@ -376,16 +376,59 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <system>, where repairs are needed from Unfettered attacks."
```

Comment:
Why `<system>` instead of `<destination>`?

Reply:
fixed